sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -16,13 +16,14 @@
     <RequiredTargetFrameworks>net461;netstandard2.0</RequiredTargetFrameworks>
 
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
-    <LangVersion>7.3</LangVersion>
+    <LangVersion>preview</LangVersion>
```

Comment:
Are the features you want not part of latest? Are they still in preview?

Reply:
Yes, IAsyncEnumerable is in preview or at least that's what compiler claimed when I tried to use them.